app/models/configuration_script.rb

```diff
@@ -2,4 +2,5 @@ class ConfigurationScript < ConfigurationScriptBase
   def self.base_model
     ConfigurationScript
   end
+  belongs_to :manager, :class_name => "ManageIQ::Providers::AutomationManager", :inverse_of => :configuration_scripts
```

Comment:
@Fryguy @blomquisg @gmcculloug I have change here `:class_name` to `ManageIQ::Providers::AutomationManager` to fix spec failure here https://travis-ci.org/ManageIQ/manageiq/jobs/405012910#L870